### PR TITLE
Add system idea

### PR DESCRIPTION
**Pull request by ccremer**
Alternative proposal of #3

### CODE REVIEW

**Comment by ccremer** [comment]
Draft: Text doesn't yet explain all components or interactions

File: docs/modules/ROOT/pages/explanation/system/idea.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.{apub2} system idea
+image::system/systemidea.drawio.svg[]
+
+* An instance of Keycloak is responsible for authoritatively store data on:
+** Organizations
+** Teams
+** Users
+
+* Data owned by Keycloak will be cached on each Zone.
```
Comment (tobru) [inline docs/modules/ROOT/pages/explanation/system/idea.adoc:14]:
```suggestion
* Data owned by Keycloak will be replicated to each Zone.
```

**Review by tobru — APPROVED** [review]
One small rewording, the rest LGTM so far. I'm not sure about everything we write here, but we're in the exploration phase so it's OK to have this currently documented so far. I'm OK going forward with the PR and closing #3.

**Comment by ccremer** [comment]
> I'm not sure about everything we write here, but we're in the exploration phase so it's OK to have this currently documented so far.

Yes, I'm fully aware that certain parts of the diagram are currently "random". It's definitely meant to be further finalized but should be a starting point.

**Pull request merged**